tests/test_checkpoint.py

```diff
@@ -17,47 +18,34 @@ def create_empty_file(path):
     path.touch()
 
 
-def test_estimator_checkpoint_creates_dir():
+def test_local_estimator_writer_creates_dir():
     prefix = "good_model"
-    with tempfile.TemporaryDirectory() as tmp_dir:
-        dirname = os.path.join(tmp_dir, "some_dir_name")
-        EstimatorCheckpoint(dirname, prefix)
+    dirname = os.path.join(const.PROJECT_ROOT, prefix)
```

Comment:
It might be better teses cleanup itself after runs. I will recommdend deleting these directory after test finished. Using `TemporaryDirectory()` helps to cleanup directory after each unittest runs.